benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -2180,6 +2180,181 @@
     "PageLabel": "Issue direction",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseEventID": "directionIssued",
+    "CaseFieldID": "confidentialityType",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "1.0",
+    "PageLabel": "Issue direction"
+  },
+  {
+    "CaseEventID": "directionIssued",
+    "CaseFieldID": "hasRepresentative",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 99,
+    "PageID": "1.0",
+    "FieldShowCondition": "confidentialityType=\"DUMMY_VALUE_TO_HIDE_FIELD\"",
```

Comment:
What is the purpose of this?

Reply:
need the hasRepresentative to be present on this event, but it should be invisible to the user. Its the only way I could do it. Without this, the bootWithCCD does not start up.

Reply:
This is very similar to issueGenericLetter event.